diff -ur a/web/menu.jspf b/web/menu.jspf
--- a/web/menu.jspf	2010-05-11 10:10:08.000000000 +0200
+++ b/web/menu.jspf	2011-02-17 13:21:20.000000000 +0100
@@ -70,10 +70,11 @@
 function selectAllProjects() {     
      $("#project *").attr("selected","selected");
 }
-function invertAllProjects() {    
+function codeOnlyProjects() {    
+     $("#project *").attr("selected","selected");
      $("#project *").each( function(){
-         if ( $(this).attr("selected") ) { $(this).removeAttr("selected") }
-         else { $(this).attr("selected","true"); }
+         var value = $(this).attr("value");
+         if ( value == 'l10n' || value == 'testing' || value == 'help' ) { $(this).removeAttr("selected") }
      }
     );
 }
@@ -101,7 +102,7 @@
                             <table border="0">
                                 <tbody>
                                     <tr>
-                                        <td>in project(s): <span style="padding-left:60px;font-size:small"><a tabindex="6" href="#" onclick="javascript:selectAllProjects();return false;">select all</a>&nbsp;|&nbsp;<a href="#" tabindex="7" onclick="javascript:invertAllProjects();return false;">invert selection</a></span></td>
+                                        <td>in project(s): <span style="padding-left:60px;font-size:small"><a tabindex="6" href="#" onclick="javascript:selectAllProjects();return false;">select all</a>&nbsp;|&nbsp;<a href="#" tabindex="7" onclick="javascript:codeOnlyProjects();return false;">all code</a></span></td>
                                     </tr>
                                     <tr><td>
                                             <select tabindex="8" onkeyup="javascript:checkEnter(event);" class="q" style="width: 300px" id="project" name="project" multiple="multiple" size="<%=Math.min(6, env.getProjects().size())%>">
